### ISSUE TITLE
Replace `rust-condition-script-cargo_make_current_task_initial_makefile_directory-substitution` plugin with new cargo make feature

### ISSUE THREAD

**Issue by wmmc88** [microsoft/windows-drivers-rs#171]
With resolution of https://github.com/sagiegurari/cargo-make/issues/1081, this plugin may be no longer needed. This can be addressed in a followup

_Originally posted by @wmmc88 in https://github.com/microsoft/windows-drivers-rs/pull/143#discussion_r1621435724_

**Comment by svasista-ms** [comment]
@wmmc88, I'm working on this issue. I'll raise a PR after #186 is merged.

**Comment by wmmc88** [comment]
@svasista-ms On closer inspection, I believe this task is blocked by https://github.com/sagiegurari/cargo-make/pull/1132 merging. While https://github.com/sagiegurari/cargo-make/issues/1081 was resolved, it still fails to expand the args based of local testing. https://github.com/sagiegurari/cargo-make/pull/1132 should fix that.

**Comment by wmmc88** [comment]
@svasista-ms you should be unblocked now
